### DISCUSSION TITLE
Details for Subtasks in Logs

### DISCUSSION THREAD

**Discussion by leopold58** [apache/airflow#21039] [General]
### Apache Airflow version

2.2.3 (latest released)

### What happened

When I want to execute shell command through PythonOperator, I chose "subprocess" to implement it.

The same code, in airflow 1.X, I can see the details, but in airflow 2.2, it's gone.

1.X
![image](https://user-images.githubusercontent.com/44051935/149468209-0128a1f8-638e-4ebd-9305-5018616af344.png)
2.X
![image](https://user-images.githubusercontent.com/44051935/149468334-a93e6bf9-43a0-437f-a24b-76a90b8cb956.png)

`
def test12(command):
    import subprocess
    subprocess.call(command, shell=True)

test_task = PythonOperator(
    task_id="test",
    python_callable=test12,
    op_args=[command],
    dag=dag
)
test_task
`

### What you expected to happen

In airflow 1.X, it execute PythonOperator with "base_task_runner.py"; in airflow 2.2, it execute PythonOperator with "standard_task_runner.py"

There are little differences between “base_task_runner.py” and "standard_task_runner.py"

### How to reproduce

I hope it‘ll be available for the details of subtasks 
or what should i do to implement it.

### Operating System

PythonOperator

### Deployment

Docker-Compose

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
Not sure what you try to do and the problem you have. I will convert into discussion or maybe you can open a PR showing what you want to dol
